examples/simple-dapp/package.json

```diff
@@ -10,6 +10,7 @@
     "deploy": "sst deploy"
   },
   "dependencies": {
+    "@radixdlt/babylon-gateway-api-sdk": "*",
```

Comment:
add the dependency by running in project root
`npm install @radixdlt/babylon-gateway-api-sdk --workspace=simple-dapp`